### PR TITLE
Create login page view, route, and controller.

### PR DESCRIPTION
**Pull request by tmartin1**

### CODE REVIEW

File: client/app/account/login/login.controller.js [hunk 1]
```diff
@@ -1,6 +1,35 @@
 'use strict';
 
 angular.module('badgerApp')
-  .controller('LoginCtrl', function ($scope) {
-    $scope.message = 'Hello';
+  .controller('LoginCtrl', function ($scope, Auth, $location) {
+    $scope.user = {};
```
Comment (scottrice10) [inline client/app/account/login/login.controller.js:5]: Indentation in lines 6 - 14 is off.

**Comment by tmartin1** [comment]
Corrected spacing, all tabs are now spaces.

**Comment by kxprim** [comment]
If you can make Travis pass we'll merge in your request.

**Comment by tmartin1** [comment]
Fixed issues in karma tests, all tests now passing locally and with travis.

**Pull request merged**